PROTOCOL.md

```diff
@@ -2,6 +2,7 @@
 <!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 # Delta Transaction Log Protocol
 
+- [Delta Transaction Log Protocol](#delta-transaction-log-protocol)
```

Comment:
nit: can you delete this line? or, is this a markdown convention I'm not aware of?